creativecloud/scripts/utils.js

```diff
@@ -86,9 +98,21 @@ function getDecorateAreaFn() {
         import(`${getConfig().codeRoot}/deps/interactive-marquee-changebg/changeBgMarquee.js`);
         break;
       }
-      case firstBlock?.classList.contains('marquee'):
-        firstBlock.querySelectorAll('img').forEach(eagerLoad);
+      case firstBlock?.classList.contains('marquee'): {
+        // Load image eagerly for specific breakpoint
+        const viewport = defineDeviceByScreenSize();
+        const lcpImageByViewport = {
+          MOBILE: 'div img',
+          TABLET: 'div:nth-child(2) img',
+          DESKTOP: 'div:nth-child(3) img',
+        };
+        const backgroundImages = firstBlock.querySelector('div');
+        if (backgroundImages.querySelectorAll('img').length === 1) eagerLoad(firstBlock.querySelector('div img'));
+        else eagerLoad(firstBlock.querySelector(lcpImageByViewport[viewport]));
+        // Foreground image
+        eagerLoad(firstBlock.querySelector('div:nth-child(2) > div img'));
         break;
+      }
```

Comment:
Since there is no block scope required specifically we can avoid curly brackets

Reply:
This was added for lint issue, hence it's needed.